pytorch/pytorch-engine/src/main/java/ai/djl/pytorch/engine/PtSymbolBlock.java

```diff
@@ -67,6 +71,19 @@ public PtSymbolBlock(PtNDManager manager, long handle) {
         first = true;
     }
 
+    /**
+     * Constructs an Empty {@code PtSymbolBlock}.
+     *
+     * @param manager the manager to use for the block
+     */
+    public PtSymbolBlock(PtNDManager manager) {
```

Comment:
This constructor is not been used, any plan to use this?

Reply:
Yes, in the blockfactory